iothub/device/src/Common/CommonResources.cs

```diff
@@ -24,5 +27,14 @@ internal static string GetString(string value, params object[] args)
 
             return value;
         }
+
+        internal static string GetNewStringGuid(string postfix)
```

Comment:
>postfix [](start = 55, length = 7)

(string postfix = "")

since most instances just pass "" into this method

Reply:
Good point - it goes to the next check-in

---
In reply to: [252001564](https://github.com/Azure/azure-iot-sdk-csharp/pull/770#discussion_r252001564) [](ancestors = 252001564)